locales/nb/pagination.php

```diff
@@ -1,17 +1,17 @@
 <?php
 
-/*
-|--------------------------------------------------------------------------
-| Pagination Language Lines
-|--------------------------------------------------------------------------
-|
-| The following language lines are used by the paginator library to build
-| the simple pagination links. You are free to change them to anything
-| you want to customize your views to better match your application.
-|
-*/
-
 return [
-    'next'     => 'Neste &raquo;',
+    /*
+    |--------------------------------------------------------------------------
+    | Pagination Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are used by the paginator library to build
+    | the simple pagination links. You are free to change them to anything
+    | you want to customize your views to better match your application.
+    |
+    */
+
     'previous' => '&laquo; Forrige',
+    'next'     => 'Neste &raquo;',
```

Comment:
why set the order ?

Reply:
Compared to and changed according to the English file.